dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/connector/teradata/MetadataQueryGenerator.java

```diff
@@ -67,6 +68,25 @@ static String createSelectForDatabasesV(
         .serialize();
   }
 
+  static String createSimpleSelect(String tableName, Optional<Expression> condition) {
+    FromClauseStepBuilder partialQuery = select("%s").from(tableName);
+    return condition.map(partialQuery::where).orElse(partialQuery).serialize();
+  }
+
+  static String createSelectForAllTempTablesVX(List<String> databases) {
+    String tableName = "DBC.AllTempTablesVX";
+    FromClauseStepBuilder builder = select("%s").from(tableName);
+    if (databases.isEmpty()) {
+      return builder.serialize();
+    }
+    return builder
+        .where(
+            in(
+                identifier("B_DatabaseName"),
+                databases.stream().map(String::toUpperCase).collect(toList())))
+        .serialize();
```

Comment:
nit: We could use existing createSimpleSelect func:
```suggestion
    return createSimpleSelect(
        tableName,
        in(identifier("B_DatabaseName"),
            databases.stream().map(String::toUpperCase).collect(toList()))
    )
    ```

Reply:
Done.